app/models/container_project.rb

```diff
@@ -28,7 +28,7 @@ class ContainerProject < ApplicationRecord
   virtual_column :containers_count,  :type => :integer
 
   def groups_count
-    container_groups.size
+    select('count(container_groups.id)')
```

Comment:
You may want to use `virtual_total` It tends to speed stuff up and we have goals to speed those up more

```ruby
virtual_total :group_counts, :container_groups
```